### DISCUSSION TITLE
Deploy to VPS

### DISCUSSION THREAD

**Discussion by 2braincells2go** [anse-app/anse#98] [Q&A]
Does anyone know of a good tutorial on how to deploy Anse to a VPS?

I am not strong on Node and before setting this up for a classroom project, like to learn more about deploying Anse to VPS. Since there will be questions from students, like to educate myself on the process :)

Thanks in advance.

**Comment by ddiu8081** [comment]
You can check out deployment options on https://docs.anse.app/self-deploy.

For deploying with Node, you can follow the steps below:

1. Pull the latest code from the VPS
2. Run `npm install` and `npm run dev`.
3. run `node dist/server/entry.mjs` (or use [pm2](https://pm2.keymetrics.io/) for process management)

Alternatively, you can run the `build` command locally and only upload the `dist` directory to the VPS.

Or you can use Docker:

Run `docker run -p 3000:3000 -d ddiu8081/anse:latest`

**Comment by 2braincells2go** [comment]
Thanks @ddiu8081 for reply.

Have been through the link provided and it shows the basics. I was hoping for little more in-depth instructions. But will go with what there is.

I am standing in for a teacher that is out on medical leave and not a server guru:)
